content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md

```diff
@@ -24,7 +24,6 @@ software:
 
 * [Portenta X8](https://store.arduino.cc/products/portenta-x8) board.
 * Wi-Fi AP with Internet access.
-* Arduino Pro Cloud Subscription. [Learn more about the Pro Cloud](https://www.arduino.cc/pro/hardware/product/portenta-x8#pro-cloud).
```

Comment:
Should not we add something pointing to that link somewhere else?

Reply:
It's already mentioned in the Register to Foundries Factory section :)